static/src/components/MilitaryUpdate.js

```diff
@@ -0,0 +1,160 @@
+import React, { PropTypes } from 'react';
+import { Card, CardMedia, CardText } from 'material-ui';
+
+function search(searchID, allPlayers) {
+    const myArray = allPlayers;
+    for (var i = 0; i < myArray.length; i++) {
+        if (myArray[i].id === searchID) {
+            return myArray[i];
+        }
+    }
+    return null;
+}
+
+function getNeighbourArmy(allData) {
```

Comment:
Do you want to put this into a seperate helper function as it is used in GameScreen too?

Reply:
OK. These two function have been moved to Military screen, as I send whole game data to Military screen to be accessed by Play.js also (for end of game updates.

WIll delete these from Gamescreen